src/data/equiv/basic.lean

```diff
@@ -1446,7 +1370,7 @@ end
 /-- The `pi`-type `Π i, π i` is equivalent to the type of sections `f : ι → Σ i, π i` of the
 `sigma` type such that for all `i` we have `(f i).fst = i`. -/
 def pi_equiv_subtype_sigma (ι : Type*) (π : ι → Type*) :
-  (Πi, π i) ≃ {f : ι → Σi, π i | ∀i, (f i).1 = i } :=
+  (Πi, π i) ≃ {f : ι → Σi, π i // ∀i, (f i).1 = i } :=
```

Comment:
This change is #9559 which is already being staged, it will go away when bors rebases.